src/plugins/notebook/components/Notebook.vue

```diff
@@ -115,7 +137,7 @@ import { clearDefaultNotebook, getDefaultNotebook, setDefaultNotebook, setDefaul
 import { addNotebookEntry, createNewEmbed, getEntryPosById, getNotebookEntries, mutateObject } from '../utils/notebook-entries';
 import objectUtils from 'objectUtils';
 
-import { throttle } from 'lodash';
+import _ from 'lodash';
```

Comment:
lets import 'debounce' instead of whole loadash,   `import { debounce } from lodash;`

Reply:
Good call, done!